pytket/pytket/circuit/__init__.py

```diff
@@ -66,15 +68,28 @@ def overload_add_wasm(
     list_i: List[List[int]],
     list_o: List[List[int]],
     args: Union[List[int], List[Bit]],
+    **kwargs: KwargTypes,
 ) -> Circuit:
     """Add a classical function call from a wasm file to the circuit.
     \n\n:param funcname: name of the function that is called
     \n:param filehandler: wasm file handler to identify the wasm file
     \n:param list_i: list of the number of bits in the input variables
     \n:param list_o: list of the number of bits in the output variables
     \n:param args: vector of circuit bits the wasm op should be added to
+    \n:param kwargs: additional information to make the wasm op conditional
     \n:return: the new :py:class:`Circuit`"""
 
+    if kwargs is not None:
```

Comment:
No need for this; `kwargs` cannot be `None` (can be empty dictionary).

Reply:
removed